store/sqlstore/supplier.go

```diff
@@ -112,13 +114,20 @@ type SqlSupplier struct {
 	stores         SqlSupplierStores
 	settings       *model.SqlSettings
 	lockedToMaster bool
+	cacheProvider  cache.CacheProvider
 }
 
-func NewSqlSupplier(settings model.SqlSettings, metrics einterfaces.MetricsInterface) *SqlSupplier {
+func NewSqlSupplier(settings model.SqlSettings, metrics einterfaces.MetricsInterface, cacheProvider cache.CacheProvider) *SqlSupplier {
```

Comment:
The SQL store doesn't need to know anything about the cache at all, I would expect this field to be in the localcachelayer.

Reply:
Done